src/config.js

```diff
@@ -46,128 +46,130 @@ const ALL_TOPICS = '*';
 /**
  * @typedef {object} PrebidConfig
  *
- * @property {bool} usePrebidCache True if we should use prebid-cache to store video bids before adding
- *   bids to the auction, and false otherwise. **NOTE** This must be true if you want to use the
- *   dfpAdServerVideo module.
+ * @property {bool} config.cacheUrl Set a url if we should use prebid-cache to store video bids before adding
```

Comment:
Looks like it should be a `{string}`

Reply:
I'm cool with that.  I'll update it to be `cache.url`